src/core/app/services/employee.service.ts

```diff
@@ -66,4 +66,23 @@ async function getAllEmployees(): Promise<EmployeeEntity[]> {
   return await EmployeeRepository.findAll();
 }
 
-export const EmployeeService = { signIn, getAllEmployees };
+export interface userToken {
+  email: string;
+  deviceToken: string;
+}
+
+/**
+ * @brief Function that saves the token of the employee
+ *
+ * @param body: userToken
+ */
+
```

Comment:
GH marca este error: 

Property 'saveToken' does not exist on type '{ create: (entity: EmployeeEntity) => Promise<EmployeeEntity>; findAll: () => Promise<EmployeeEntity[]>; findByEmail: (email: string) => Promise<...>; findById: (id: string) => Promise<...>; existByEmail: (email: string) => Promise<...>; updateRoleById: (id: string, roleId: string) => Promise<...>; }'.

Reply:
listo, se pusheó la solución